Form/Type/ButtonIconType.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace SumoCoders\FrameworkExampleBundle\Form\Type;
+
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\FormBuilderInterface;
+
+class ButtonIconType extends AbstractType
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function buildForm(FormBuilderInterface $builder, array $options)
+    {
+        $this->addButtonIconSection($builder, $options);
+    }
+
+    private function addButtonIconSection($builder, array $options)
```

Comment:
why create a separate method for this? I'd just add this in the buildForm method.